services/crates/crates-base.js

```diff
@@ -9,7 +9,7 @@ const keywords = ['Rust']
 const crateSchema = Joi.object({
   crate: Joi.object({
     downloads: nonNegativeInteger,
-    recent_downloads: nonNegativeInteger,
+    recent_downloads: nonNegativeInteger.allow(null),
```

Comment:
Instead of putting the default value in the transform, can you use `.default(0)` here?

Reply:
That only works for `undefined`, not for `null`; I tried that :(

Reply:
> That only works for undefined, not for null

I didn't realize that was the case either but the [docs](https://hapi.dev/module/joi/api/?v=17.1.1#anydefaultvalue) seem to support that